proxy/src/test/java/com/wavefront/agent/listeners/otlp/OtlpMetricsUtilsTest.java

```diff
@@ -860,4 +866,92 @@ public void appliesPreprocessorRules() {
 
     assertAllPointsEqual(expectedPoints, actualPoints);
   }
+
+  @Test
+  public void testUpdateAttrsList() {
```

Comment:
Can we find a more description test name, since we aren't updating (mutating) a list of attrs? Perhaps just `TestAppTagsFromResourceAttrs()`.

Reply:
Sure